lib/stripe.rb

```diff
@@ -384,6 +384,12 @@ def self.handle_restclient_error(e, request_opts, retry_count, api_base_url=nil)
       message = "The connection to the server (#{api_base_url}) broke before the " \
         "request completed. #{connection_message}"
 
+    when OpenSSL::SSL::SSLError
+      message = "Could not establish a secure connection to Stripe, you may " \
+                "need to upgrade your OpenSSL version. To check, try running " \
+                "'openssl s_client -connect api.stripe.com:443' from the " \
+                "command line."
```

Comment:
So any error message raised by OpenSSL is generally useless, but what do you think about tacking it on  in here somewhere anyway? Usually if you're debugging some particularly annoying problem you'll take whatever you can get.